recipes-gnome/librsvg/librsvg/0002-allow-invalid-ref-casting-to-allow-building.patch

```diff
@@ -0,0 +1,20 @@
+--- librsvg-2.52.10/src/surface_utils/mod.rs	2024-10-15 20:59:07.374334093 +0200
++++ librsvg-2.52.10/src/surface_utils/mod.rs_mod	2024-10-15 20:58:56.391383870 +0200
+@@ -147,12 +147,12 @@
+     #[inline]
+     fn set_pixel(&mut self, stride: usize, pixel: Pixel, x: u32, y: u32) {
+         let value = pixel.to_u32();
++        let index = (y as usize + stride) + (x as usize * 4);
```

Comment:
Shouldn't this be `y as usize * stride`?

Reply:
Oh, yes you are absolutely correct, this was a typo.